### ISSUE TITLE
Unsupported device: 0x4e2a Error

### ISSUE THREAD

**Issue by Stefos13** [mjg59/python-broadlink#580]
Hello, I have an RM Mini and I'm getting the following error. The commands are being sent without any problems and the device works. Everything is updated to the latest version.

> Logger: homeassistant.components.broadlink.config_flow
Source: components/broadlink/config_flow.py:41
Integration: Broadlink (documentation, issues)
First occurred: 12:51:23 PM (1 occurrences)
Last logged: 12:51:23 PM
Unsupported device: 0x4e2a. If it worked before, please open an issue at https://github.com/home-assistant/core/issues

**Comment by KTibow** [comment]
Which type of RM Mini?

**Comment by Stefos13** [comment]
It's Rm Mini 3.. I don;t know if you ask for something else. If yes where can i find the type of the rm mini?

**Comment by KTibow** [comment]
Reboot HA, then see if it's still there.

**Comment by Stefos13** [comment]
I have done it many times.. Still there

**Comment by felipediel** [comment]
This error message will disappear after https://github.com/home-assistant/core/pull/49394.

We don't have 0x4e2a in our code base. What device is this?

**Comment by Stefos13** [comment]
> This error message will disappear after [home-assistant/core#49394](https://github.com/home-assistant/core/pull/49394).
> 
> We don't have 0x4e2a in our code base. What device is this?

The device is the Broadlink RM Mini 3. Maybe i can check the device itself later to see if i can find anything extra

**Comment by felipediel** [comment]
Interesting, I had never seen an RM mini 3 in this range. Forget what I said above. Could you please confirm that it works with the rmminib class?

```python3
import broadlink as blk
d = blk.rmminib(('192.168.0.16', 80), 'aabbccddeeff', 0x4e2a)
d.auth()
d.send_data(bytes(range(255)))
```

Does the LED blink?

**Comment by Stefos13** [comment]
> Interesting, I had never seen an RM mini 3 in this range. Forget what I said above. Could you please confirm that it works with the rmminib class?
> 
> ```python
> import broadlink as blk
> d = blk.rmminib(('192.168.0.16', 80), 'aabbccddeeff', 0x4e2a)  # Example IP and MAC address
> d.auth()
> d.send_data(bytes(range(255)))
> ```
> 
> Does the LED blink?

I will try it once i get home but I need you to tell me how to try this. Just run it as python script from any device in my network I guess, and the IP of the device.

**Comment by felipediel** [comment]
Yes.

**Comment by Stefos13** [comment]
I tried it with Python 3.7 and i get this error 
"Traceback (most recent call last):
  File "t.py", line 2, in <module>
    d = blk.rmminib(('192.168.1.4', 80), 'aabbccddeeff', 0x4e2a)  # Example IP and MAC address
AttributeError: module 'broadlink' has no attribute 'rmminib'"

Don;t know if I am missing something or the problem is somewhere else..

**Comment by felipediel** [comment]
Your library is not up to date. Try using the rm4 class.

**Comment by Stefos13** [comment]
Haven't managed to make it work yet.. I have installed broadlink etc and I'm getting "ImportError No module named broadlink"
I will try to setup and VM with linux and do it from there

**Comment by Stefos13** [comment]
I installed ubuntu , installed broadlink and im getting this error now
![Capture](https://user-images.githubusercontent.com/18400828/115445552-ae2f8300-a21e-11eb-8b2d-409c7cf719e3.JPG)

**Comment by felipediel** [comment]
Ok, this is not an rm4. Let's try the rm class:
```python3
import broadlink as blk
d = blk.rm(('192.168.0.16', 80), 'aabbccddeeff', 0x4e2a)  # Example IP and MAC address
d.auth()
d.send_data(bytes(range(255)))
```

**Comment by Stefos13** [comment]
Ok that worked, the light blinked!

**Comment by felipediel** [comment]
Awesome. I am creating a PR right now.

**Comment by felipediel** [comment]
Done. If you want to patch your installation before the release:

1. Open Home Assistant and install "SSH & Web Terminal" add-on
2. Disable Protection mode
3. Click Terminal on the sidebar and install the patch:
```
docker exec -it homeassistant /bin/bash
pip3 install git+https://github.com/felipediel/python-broadlink.git@0x4e2a --force-reinstall --no-deps
```
4. Restart the server.

You need to reinstall the patch every time you update Home Assistant.

**Comment by felipediel** [comment]
Just to confirm... Is [this](https://www.amazon.com.br/BroadLink-RM-Mini-Compat%C3%ADvel-Assistente/dp/B01FK2SDOC/ref=asc_df_B01FK2SDOC/?tag=googleshopp00-20&linkCode=df0&hvadid=379728501964&hvpos=&hvnetw=g&hvrand=2494668768212166995&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1001686&hvtargid=pla-318070470702&psc=1) your device?

**Comment by Stefos13** [comment]
> Just to confirm... Is [this](https://www.amazon.com.br/BroadLink-RM-Mini-Compat%C3%ADvel-Assistente/dp/B01FK2SDOC/ref=asc_df_B01FK2SDOC/?tag=googleshopp00-20&linkCode=df0&hvadid=379728501964&hvpos=&hvnetw=g&hvrand=2494668768212166995&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1001686&hvtargid=pla-318070470702&psc=1) your device? I checked all the apps (Broadlink, IHC and e-Control) and the device is not there. Are you sure about the type?
> 
> ```python
> import broadlink as blk
> d = blk.hello("192.168.0.16")  # Example IP address
> print(d)
> ```
> 
> What is the output?

Yeap, that's the device. I will try it later because i am at work now and i will let you know

**Comment by Stefos13** [comment]
> Just to confirm... Is [this](https://www.amazon.com.br/BroadLink-RM-Mini-Compat%C3%ADvel-Assistente/dp/B01FK2SDOC/ref=asc_df_B01FK2SDOC/?tag=googleshopp00-20&linkCode=df0&hvadid=379728501964&hvpos=&hvnetw=g&hvrand=2494668768212166995&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1001686&hvtargid=pla-318070470702&psc=1) your device? I checked all the apps (Broadlink, IHC and e-Control) and the device is not there. Are you sure about the type?
> 
> ```python
> import broadlink as blk
> d = blk.hello("192.168.0.16")  # Example IP address
> print(d)
> ```
> 
> What is the output?

Here you are
智能遥控 (Broadlink RM mini 3 0x2737 / 192.168.)

**Comment by felipediel** [comment]
Your RM mini 3 is a 0x2737. As I suspected, 0x4e2a is a different device with Broadlink DNA. Maybe an AC unit?

**Comment by danielchalmers** [comment]
> Closing, if you figure out what device is 0x4e2a, please let us know! Thank you!

@felipediel you're right, 0x4e2a is indeed an AC unit. I found the same code for mine

**Comment by lastsoft81** [comment]
hello.
so I came across this thread as im looking to connect my AUX AC UNIT to my homeassitant with no luck.
it has the same id 0x4e2a and HA says not supported when im trying to add it.
is is possible to add this ac unit to HAOS?


Best regards,
Shlomi Chen.
